elibrary-hibernate/src/main/resources/db-changelog-master.xml

```diff
@@ -13,7 +13,7 @@
 
 	<!-- Pre-processing -->
 	<!-- Increment id of changeSet -->
-	<include file="src/main/resources/db/changelog/truncate-all-cascade-changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
Please remove it as it's a local change. 🗑️

Reply:
I did it.